cypress.config.ts

```diff
@@ -17,6 +17,16 @@ export default defineConfig({
       on('task', {
         reset: resetStubs,
         ...auth,
+        log(message) {
+          // eslint-disable-next-line no-console
+          console.log(message)
```

Comment:
Can we not use logger instead of console.log?

Reply:
This is actually intentional, so you can see data in the console log on a test step.

Reply:
It does not modify `cy.log()`, it is just for task() calls, e.g. https://docs.cypress.io/api/commands/task#Arguments. I have removed `log` as it was unused.